packages/terra-form-input/src/terra-dev-site/doc/example/controlled/InputProgrammaticSet.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames/bind';
+import Input from 'terra-form-input';
+import styles from './InputProgrammaticSet.module.scss';
+
+const cx = classNames.bind(styles);
+
+const InputProgrammaticSet = () => {
+  const [text, setText] = useState('');
+  const inputRef = useRef(null);
+
+  const setTextClick = () => {
+    if (inputRef.current) {
+      inputRef.current.value = 'IpsumLorem';
+      let customEvent;
+      if (typeof (Event) === 'function') {
+        customEvent = new Event('input', {
+          bubbles: true,
+          cancelable: false,
+        });
+      } else {
+        customEvent = document.createEvent('Event');
+        customEvent.initEvent('input', true, false);
```

Comment:
[initEvent](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent) has been deprecated. The suggestion is to use the event constructor.

Reply:
Updated here https://github.com/cerner/terra-core/pull/3135/commits/d11f3adca5083f5aa752a496fd8f7dc00e6b8a31